src/python/WMComponent/JobCreator/JobCreatorPoller.py

```diff
@@ -109,6 +110,10 @@ def capResourceEstimates(jobGroups, constraints):
             j['estimatedJobTime'] = min(j['estimatedJobTime'], constraints['MaxWallTimeSecs'])
             j['estimatedDiskUsage'] = min(j['estimatedDiskUsage'], constraints['MaxRequestDiskKB'])
 
+            # finally, quantize those
+            j['estimatedJobTime'] = quantize(j['estimatedJobTime'], constraints['MinWallTimeSecs'])
```

Comment:
j['estimatedJobTime'] = quantize(j['estimatedJobTime'], constraints['MinWallTimeSecs'])

is quantizing the estimatedJobTime by MinWallTimeSecs= 1 * 3600 , 1h ; sounds a bit steep. Why not adding further parameters WallTimeSecsQuanta ?

Reply:
I just wanted to make this code more readable and maintainable. 1h seems quite reasonable to me. Maybe a feedback from the SI team would be better @aperezca @bbockelm

Reply:
Given how much of an approximation all this scheduling is, 1 hour is probably as accurate as I would believe!  I would also accept 2 hours... I suspect there's decreasing returns for higher values.